helm/docs/upgrading.md

```diff
@@ -143,7 +143,7 @@ quickstart  0.21.2
 
 <!-- Differs from docker-compose docs -->
 
-In `v2.10.2`, the `values.yaml` location of startup probe settings
+In `v2.9.0`, the `values.yaml` location of startup probe settings
```

Comment:
This and the one below seemed like a past script search issue - it should say v2.9.0 here I believe